CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
 # Changelog
+## [7.0.0] - 2022-12-25
+- Added support for generating drift tables by adding `generate_table: true` on model definition
```

Comment:
Use different name to tie this to drift

Reply:
I specifically didn't want this in case we (or someone else) adds a converter for another database package 🤔

Reply:
And then they have a name for this one. We can keep the 'primary_key' names etc to be reused.
Else we would beed a: generate tables: true, flavor: drift. Which would be the same as: generate_drift_tables